README.md

```diff
@@ -410,6 +410,17 @@ Specifies the cluster name to tag allocated ENIs with. See the "Cluster Name tag
 
 ---
 
+#### `CLUSTER_ENDPOINT`
```

Comment:
Nit: Lets mention the release version, CLUSTER_ENDPOINT (v1.12.1+)

Reply:
updated!